tests/server/test_compilerservice.py

```diff
@@ -843,8 +841,14 @@ async def old_and_new_compile_report(server_with_frequent_cleanups, environment_
         "name": "Recompiling configuration model",
         "compile": compile_id_new,
     }
-    await Report(**report1).insert()
-    await Report(**report2).insert()
+
+    async with Compile.get_connection() as con:
```

Comment:
I'm not 100% sure about this line.
Is this an ok way to retrieve the connection ?

Reply:
Looks good to me. I don't think we currently have a more generic way to retrieve the connection.